Userland/Libraries/LibGL/SoftwareGLContext.h

```diff
@@ -39,6 +40,8 @@ struct ContextParameter {
 };
 
 class SoftwareGLContext : public GLContext {
+    static constexpr GLint max_lights = 8;
```

Comment:
It might make more sense to put this in the device info from `LibSoftGPU`, since it will probably be the graphics device determining how many lights it can handle.

Reply:
Good point. I've actually got it in there already (via the static config), so it's actually redundant. I'll get this by going through the device instead.